health_check.sh

```
@@ -58,4 +58,73 @@ else
+hostport="${userpass_hostport#*@}"
+
+unset RABBITMQ_USER
+unset RABBITMQ_PASS
+unset RABBITMQ_HOST
+unset RABBITMQ_PORT_AMQP
+unset VHOST
+
+RABBITMQ_USER="${userpass%%:*}"
+RABBITMQ_PASS="${userpass#*:}"
+RABBITMQ_HOST="${hostport%%:*}"
+RABBITMQ_PORT_AMQP="${hostport#*:}"
```

Shellcheck (0.10.0)

[warning] RABBITMQ_PORT_AMQP appears unused. Verify use (or export if used externally).

(SC2034)